cloudify_cli/execution_events_fetcher.py

```diff
@@ -72,8 +64,29 @@ def _fetch_events(self):
             raise
         return events
 
-    def fetch_all(self):
-        return self.fetch_events(get_remaining_events=True)
+    def fetch_and_process_events(self, events_handler=None):
+        total_events_count = 0
+        deadline = time.time() + self._timeout
+
+        while True:
+            if time.time() > deadline:
+                raise EventProcessingTimeoutError(
+                    self._execution_id,
+                    'events/log fetching timed out for execution timed out')
+
+            events_batch_count = self._fetch_and_process_events_batch(
+                events_handler=events_handler)
+
+            total_events_count += events_batch_count
+
+            if events_batch_count < self._batch_size:
+                # returned less events than allowed by _batch_size,
+                # this means these are the last events
+                break
+
+            time.sleep(1)
+
+        return total_events_count
 
 
 def get_all_execution_events(client, execution_id, include_logs=False):
```

Comment:
Is anyone calling get_all_execution_events? Because this function calls execution_events.fetch_all() which is removed in the PR